tests/test_elasticity.py

```diff
@@ -4,17 +4,38 @@
 import pytest
 
 from matcalc.elasticity import ElasticityCalc
+import numpy as np
 
 
-def test_elastic_calc(LiFePO4, M3GNetCalc):
+def test_elastic_calc(Li2O, M3GNetCalc):
```

Comment:
What's the reason for changing test structure?

Reply:
It seems like the LiFePO4 is returning negative bulk modulus values for both M3GNet and CHGNet, which seems to be a result of the stress values:

```
>>> m3gnet.get_stress(atoms = adaptor.get_atoms(LiFePO4()))
array([[ 6.1987960e-03, -1.2431070e-03,  2.1655680e-06],
       [-1.2431042e-03,  4.7054393e-03,  6.8889981e-06],
       [ 2.1623537e-06,  6.8931222e-06,  5.5809757e-03]], dtype=float32)
>>> m3gnet.get_stress(atoms = adaptor.get_atoms(LiFePO4().copy().apply_strain(0.02)))
array([[-4.0997554e-02, -7.1016338e-04,  1.4348395e-06],
       [-7.1014854e-04, -3.8529243e-02,  4.4988519e-06],
       [ 1.4608503e-06,  4.4946100e-06, -4.4417910e-02]], dtype=float32)
>>> chgnet.get_stress(atoms = adaptor.get_atoms(LiFePO4()))
array([[ 5.1067616e-03, -1.0610512e-03,  3.1236543e-06],
       [-1.0610349e-03,  2.7534107e-03,  1.0496358e-05],
       [ 3.1559418e-06,  1.0493268e-05,  5.1379004e-03]], dtype=float32)
>>> chgnet.get_stress(atoms = adaptor.get_atoms(LiFePO4().copy().apply_strain(0.02)))
array([[-4.2596061e-02, -5.1366811e-04,  1.9736901e-06],
       [-5.1366567e-04, -4.0909901e-02,  7.1563909e-06],
       [ 1.9491683e-06,  7.1602840e-06, -4.4081591e-02]], dtype=float32)
```

I'm not sure why these stress values are changing in the opposite direction that they should (I have not seen this with any of the structures I've tested), so I've replaced it with Li2O, which behaves more normally. 

(To be clear, originally, the code sometimes returned a bulk modulus much higher than the value that is supposed to be predicted from the calculator, because the relaxation with M3GNet was able to (incorrectly) send the structure to a stress lower than the surrounding structures, likely due to a problem with M3GNet. I believe that problem has been recently fixed, but before that occurred, I also reimplemented ElasticityCalc to work around the issue by discarding the equilibrium structure and using multiple points for the fitting instead of just two.)

In any case, the Li2O calculation is also much cheaper, and I cannot run the LiFePO4 relaxation without getting an OOM on my laptop.

Reply:
Thanks for the detailed reply! 👍 That all sounds good. If you can fix the failing test and address @shyuep's comment re backwards compatibility for single floats, I think it's ready to go.